playbooks/adhoc/uninstall.yml

```diff
@@ -347,8 +347,7 @@
     - /etc/systemd/system/etcd_container.service
 
   - name: Remove etcd data
-    shell: rm -rf /var/lib/etcd/*
-    failed_when: false
+    file: path=/var/lib/etcd/ state=absent
```

Comment:
Yes, this is intentional because if someone had mounted a filesystem at /var/lib/etcd then the contents was not removed correctly. Perhaps set `warn: no` ?